var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -52,7 +52,7 @@ class Llvm(Package):
     depends_on('cmake @2.8.12.2:')
 
     # Universal dependency
-    depends_on('python@2.7:')
+    depends_on('python@2.7:2.8')  # Seems not to support python 3.X.Y
```

Comment:
`@2.7` is `@2.7.0`. You want to allow `@2.7.11` as well. I'd write `@2.7:2.7.999`.